Library/Homebrew/cmd/mark.rb

```diff
@@ -0,0 +1,101 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "abstract_command"
+require "formula"
+require "tab"
+
+module Homebrew
+  module Cmd
+    class Mark < AbstractCommand
```

Comment:
I'm not 👎🏻 on the name but I'm not 👍🏻 either. `mark` feels like quite a broad term that I wonder if we'd want to use on anything else in future? @Homebrew/maintainers any thoughts on this,

Reply:
I don't hate the name either but you're right that it's not very specific. Maybe `brew receipt` or `brew install-receipt` instead? I can't think of a more specific verb that would work here.

Reply:
`brew tab` could also work if we're going down the receipt route.

Reply:
Seems we've encountered one of the two hard problems in computer science. :D

`mark` first came into my mind because I've been using [`apt-mark`](https://manpages.ubuntu.com/manpages/noble/man8/apt-mark.8.html) on my Debian-based systems so I think it's more intuitive. Similarly, there is [`dnf mark`](https://dnf.readthedocs.io/en/latest/command_ref.html#mark-command-label).

If we want to reserve this and be more specific, `tab` sounds better to me than `receipt` and `install-receipt` because [the former is a proper Homebrew term](https://docs.brew.sh/Formula-Cookbook#homebrew-terminology).

Reply:
`tab` sounds like a generic tab information/manipulation tool which this isn't if it only marks as installed/dependency.

Reply:
> Of course, it could expand to be but is there anything else we would want modified?

I could see it expanding beyond that. Avoiding new nouns feels nicer if possible.

Reply:
Cool, I'll go with `brew tab` then.